cmd/ddev/cmd/composer-create.go

```diff
@@ -241,3 +240,61 @@ func init() {
 	ComposerCmd.AddCommand(ComposerCreateProjectCmd)
 	ComposerCmd.AddCommand(ComposerCreateCmd)
 }
+
+// removeAll removes all files and folders in path except the ones listed in
+// skipRecursive and skip where skip only preserves the folder itself but still
+// removes containing files and folders.
+func removeAll(aPath string, skipRecursive, skip []string) error {
+	fsObjects, err := fileutil.ListFilesInDir(aPath)
+	if err != nil {
+		return err
+	}
+
+ObjectsLoop:
+	for _, fsObject := range fsObjects {
+		// Just skip objects listed in skipRecursive.
+		for _, skipObject := range skipRecursive {
```

Comment:
Maybe I'm mistaken, but can't this look just be done outside the fsObjects loop? Doesn't need to be nested for-loops?

Reply:
No, because on the next line fsObject is used in the condition.